examples/core/core_input_gamepad.nim

```diff
@@ -0,0 +1,179 @@
+#[
+*
+*   raylib [core] example - Gamepad input
+*
+*   NOTE: This example requires a Gamepad connected to the system
+*         raylib is configured to work with the following gamepads:
+*                - Xbox 360 Controller (Xbox 360, Xbox One)
+*                - PLAYSTATION(R)3 Controller
+*         Check raylib.h for buttons configuration
+*
+*   This example has been created using raylib 2.5 (www.raylib.com)
+*   raylib is licensed under an unmodified zlib/libpng license (View raylib.h for details)
+*
+*   Copyright (c) 2013-2019 Ramon Santamaria (@raysan5)
+*   Adapted in 2021 by jseb
+*
+********************************************************************************************]#
+
+import nimraylib_now
+
+# NOTE: Gamepad name ID depends on drivers and OS
+when defined(PLATFORM_RPI):
+  const XBOX360_NAME_ID = "Microsoft X-Box 360 pad"
+else:
+  const XBOX360_NAME_ID = "Xbox 360 Controller"
+
+const
+  PS3_NAME_ID = "PLAYSTATION(R)3 Controller"
+  XBOX360_LEGACY_NAME_ID = "Xbox Controller"
+  screenWidth = 800
+  screenHeight = 450
+
+type tex_names = enum PS3Pad, XboxPad
+var texs:array[tex_names,Texture2D]
+
+
+proc init() =
+  setConfigFlags(MSAA_4X_HINT)  # Set MSAA 4X hint before windows creation
+
+  initWindow(screenWidth, screenHeight, "raylib [core] example - gamepad input")
+
+  texs[Ps3Pad] = loadTexture("resources/ps3.png")
+  texs[XboxPad] = loadTexture("resources/xbox.png")
+
+  setTargetFPS( 60)
+
+proc pad_draw_xbox
+proc pad_draw_psx
+
```

Comment:
There's a lot of structure that is not present in the original implementation, we might as well use idiomatic Nim-style camelCase for variable names, e.g. here it would be `padDrawXbox`. This also applies to other places and other names

Reply:
ok